docs/clouds/qemu.md

```diff
@@ -0,0 +1,104 @@
+# QEMU
+
+The following page documents the QEMU cloud integration in pycloudlib.
+
+## Support
+
+Pycloudlib currently supports launching x86-64 Ubuntu images using
+`qemu-system-x86_64` on an x86-64 host.
+
+## Requirements
+
+In addition to requirements specified in `setup.py`, on Ubuntu systems,
+the QEMU cloud requires the following packages:
+
+* qemu-system-x86
+* qemu-utils  # for qemu-img
+* genisoimage  # to create the cloud-init NoCloud seed image
```

Comment:
Not for this PR: 
I like that we are building in supplemental utilities necessary to setup and launch vm, i wonder longer-term if we want to support use-cases where folks want to customize these images using `mount-image-callback` or `virt-customize` prior to initial launch of those images. Surely, these preparatory steps can be performed outside pycloudlib and fed into the `Qemu.launch(image_id="/my/customized/image")`, so probably doesn't make sense at this point. But, seeing our helper methods creating iso images, customizing base images before boot is a frequently used feature when dealing with qemu that we may want to offer in the future if possible.

Reply:
Yes, thanks for mentioning these tools. I tried myself with qemu-nbd and guestmount, but ran into some brick walls.